lib/replicache.dart

```diff
@@ -256,14 +256,17 @@ class Replicache implements ReadTransaction {
 
   Future<void> _sync() async {
     final syncHead = await _beginSync();
-    await _maybeEndSync(syncHead);
+    if (syncHead != '00000000000000000000000000000000') {
```

Comment:
It seems like maybe you are testing against HEAD of `repm`. Now that the version pinning is landed, you should switch to using it.